OpenRA.Mods.Common/Projectiles/LaserZap.cs

```diff
@@ -112,6 +119,8 @@ public LaserZap(LaserZapInfo info, ProjectileArgs args, Color color)
 
 			if (!string.IsNullOrEmpty(info.HitAnim))
 				hitanim = new Animation(args.SourceActor.World, info.HitAnim);
+			else
+				animationComplete = true;
```

Comment:
Can we rename this to something like `showHitAnimation` (and invert the value if needed)?  Having a variable called `animationComplete` be true when there is no animation at all isn't very clear.